lib/params/FlowParams.cpp

```diff
@@ -64,6 +52,30 @@ FlowParams::~FlowParams()
     SetDiagMsg( "FlowParams::~FlowParams() this=%p", this );
 }
 
+int FlowParams::Initialize() {
```

Comment:
This becomes interesting... RendererParams already has a private method called `_init()`. Now we introduce a public method called `Initialize()`. This design seems confusing...  Maybe one of them could be renamed to be more specific?

Reply:
RenderParams::Initialize() is public and has a documented API. RenderParams::_init() is private. They both perform initialization.